home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/BaseApiException.java

```diff
@@ -0,0 +1,8 @@
+package com.softserveinc.ita.homeproject.homeservice.exception;
+
+public abstract class BaseApiException extends RuntimeException {
```

Comment:
The service layer should know nothing about presentation so we cannot use anything from API here. From the package I see that this exception belongs to service layer, please remove API mentioning from the naming:
```
BaseHomeException
  NotFoundHomeException
  AlreadyExsistHomeException
```